package.json

```diff
@@ -4,7 +4,15 @@
   "description": "Module #2: Basics. Node.js fundamentals",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "babel-node": "babel-node --presets='@babel/preset-env' --ignore='foo|bar|baz'",
```

Comment:
no need to add presets configuration here, as you already have it in babel config file

By the way, what is the purpose of `ignore` config here?